api/controllers/FlairController.js

```diff
@@ -88,7 +88,7 @@ module.exports = {
           user.name,
           css_class,
           flair,
-          app.sub, function (err, css_class) {
+          app.sub, function (err, response) {
```

Comment:
We don't actually use this parameter for anything, but the variable name `css_class` is already used, so this was causing things to mess up.